core-bundle/src/Image/Studio/Figure.php

```diff
@@ -257,7 +257,7 @@ public function getLegacyTemplateData($margin = null, string $floating = null, b
 
         $image = $this->getImage();
         $originalSize = $image->getOriginalDimensions()->getSize();
-        $fileInfoImageSize = (new File(rawurldecode($image->getImageSrc())))->imageSize;
+        $fileInfoImageSize = (array) ((new File($image->getImageSrc(true)))->imageSize);
```

Comment:
```suggestion
        $fileInfoImageSize = (array) (new File($image->getImageSrc(true)))->imageSize;
```

I think the extra parentheses are not necessary: https://3v4l.org/imrP3